docs/index.html

```diff
@@ -680,6 +680,21 @@
 
 ---
 
+## Recent Timeline of Ops Features
+
+- 2015 1.9 Overlay (multi-host) networking, network/IPAM plugins
+- 2016 1.10 Distributed DNS
+- 2016 1.11 DNS round robin load balancing
+- 2016 1.12 Swarm mode, routing mesh, encrypted networking, healthchecks
+- 2017 1.13 Stacks, Overlay attachable, image squash and compress
+- 2017 1.13 Swarm mode for Windows Server 2016
+- 2017 17.03 Secrets
+- 2017 17.04 Update rollback, topo-aware scheduling
+- 2017 17.06 Swarm configs, multi-stage image builds, service logs
```

Comment:
I think services logs and multistage builds is 17.05; but configs might be 17.06 -- doublecheck?